elements/designer-stage/designer-stage.html

```diff
@@ -431,32 +448,33 @@
               this.insertPosition));
         }
         this._hideHover();
+        this.insertPosition = null;
         this.$.proxy.style.display = 'none';
       },
 
       _onDoubleClick(e) {
-        var bounds = this.getBoundingClientRect();
-        this.client.selectElementAtPoint(e.clientX, e.clientY, bounds)
-            .then(function(response) {
-              this._updateCurrentElement(response.elementInfo);
-              this._updateSelectionBounds(response.bounds);
-              this.enterTextMode();
-            }.bind(this));
+        let position = domUtils.toLocalPosition(this, e.clientX, e.clientY);
+        this.client.selectElementAtPoint(position.x, position.y)
+          .then(function(response) {
+            this._updateCurrentElement(response.elementInfo);
```

Comment:
This is orthogonal to this change, but `set` seems like a more appropriate verb than `update` to me. No action required, just observing.